### ISSUE TITLE
The details about "blk(x, return_attention=True)"

### ISSUE THREAD

**Issue by andytianph** [guijiejie/AMT#2]
Thank you for your excellent work and open source the code!
In the pre-training process, every 40 epochs need to update the mask weight, **"attn_map"** is obtained by **model.forward_encoder_test**,
and in the Transformer blocks of **forward_encoder_test**, finally "return blk(x, return_attention=True)" , but the original ViT block does not have a **"return_attention"** parameter, what are the specific implementation details of this part?
Looking forward to and thank you for your answer!
![1](https://user-images.githubusercontent.com/47450447/224923473-2be93f86-5393-4604-9fe9-e949c655e1f2.png)
![2](https://user-images.githubusercontent.com/47450447/224923477-8a1018dc-8b19-40d1-97a3-1fb415f4ae2e.png)
